extensions/data-workspace/src/dialogs/projectDashboard.ts

```diff
@@ -106,32 +106,85 @@ export class ProjectDashboard {
 	}
 
 	private createContainer(title: string, location: string): azdata.FlexContainer {
-		const dashboardData: IDashboardTable[] = this.projectProvider!.dashboardComponents;
-
 		const rootContainer = this.modelView!.modelBuilder.flexContainer().withLayout(
 			{
 				flexFlow: 'column',
 				width: '100%',
 				height: '100%'
 			}).component();
 
+		const headerContainer = this.createHeader(title, location);
+		const tableContainer = this.createTables();
+
+		rootContainer.addItem(headerContainer);
+		rootContainer.addItem(tableContainer);
+
+		return rootContainer;
+	}
+
+	/**
+	 * Create header with title, location and background
+	 */
+	private createHeader(title: string, location: string): azdata.Component {
+		const headerContainer = this.modelView!.modelBuilder.flexContainer().withLayout(
+			{
+				flexFlow: 'column',
+				width: '100%',
+				height: '30%'
+			}).component();
+
+		const header = this.modelView!.modelBuilder.flexContainer().withLayout(
+			{
+				flexFlow: 'column',
+				width: '100%',
+				height: '30%'
+			}).component();
+
 		const titleLabel = this.modelView!.modelBuilder.text()
 			.withProperties<azdata.TextComponentProperties>({ value: title, CSSStyles: { 'margin-block-start': '20px', 'margin-block-end': '0px' } })
 			.component();
-		rootContainer.addItem(titleLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'font-size': '36px', 'font-weight': '400' } });
+		header.addItem(titleLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'font-size': '36px', 'font-weight': '400' } });
 
 		const projectFolderPath = path.dirname(location);
 		const locationLabel = this.modelView!.modelBuilder.text()
 			.withProperties<azdata.TextComponentProperties>({ value: projectFolderPath, CSSStyles: { 'margin-block-start': '20px', 'margin-block-end': '0px' } })
 			.component();
-		rootContainer.addItem(locationLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'padding-bottom': '50px', 'font-size': '16px' } });
+		header.addItem(locationLabel, { CSSStyles: { 'padding-left': '34px', 'padding-top': '15px', 'padding-bottom': '50px', 'font-size': '16px' } });
+
+		const image = this.projectProvider!.image;		// background image added at the bottom right of the header
+		headerContainer.addItem(header, {
+			CSSStyles: {
+				'background-image': `url(${vscode.Uri.file(image!.light.toString())})`,
+				'background-repeat': 'no-repeat',
+				'background-position': '85% bottom',
+				'background-size': '10%',
+				'border-bottom': 'solid 1px',
```

Comment:
can the color match the color of the lines in the table or the toolbar border? The black looks very dark compared to the other lines in the light theme.

Reply:
it will be great if you can figure this out.

Reply:
Updated border color to match the lines in the table.